### DISCUSSION TITLE
show lable title

### DISCUSSION THREAD

**Discussion by maxiaojunwolf** [aminalaee/sqladmin#345] [Ideas]
sometimes i  need to know the label's db column name by mouse over
this is now what i do
![image](https://user-images.githubusercontent.com/34365493/192935839-cab3a6af-5e06-4aed-8535-1c7d1cea1696.png)

**Comment by aminalaee** [comment]
So you are suggesting to add `label` for create/edit forms, right?

**Comment by maxiaojunwolf** [comment]
yes,I hope there are some property like column_labels that i can set label's property&nbsp;



---Original---
From: "Amin ***@***.***&gt;
Date: Thu, Sep 29, 2022 15:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [aminalaee/sqladmin] show lable title (Discussion #345)




 
So you are suggesting to add label for create/edit forms, right?
